sites/partners/styles/overrides.scss

```diff
@@ -0,0 +1,5 @@
+:root {
+  .field-value-error {
```

Comment:
This is a temporary workaround. We have this issue where some of our `Select` and `FieldGroup` components are using the old `ViewItem` as that field(s) label. When a field is in an error state, the label also needs to turn red. It was a misuse of the component. Rather that create a new one for this interim period before those fields are in seeds, I'm just adding the error state here.